src/bin/driver.rs

```diff
@@ -482,6 +493,17 @@ mod app {
         });
     }
 
+    // This only processes the Low noise channel relay now.
+    // Todo: figure out how to handle both channels in one function.
+    #[task(priority = 1, shared=[driver_relay_state])]
+    fn handle_relay(mut c: handle_relay::Context) {
```

Comment:
I just have to solve this problem still. I would like to only have one `relay_done` rtic task handling both the LN and HP relays.

Reply:
nevermind, that was easy with just a function parameter